docs/BreakingChanges/115 AppDomainSetup.DynamicBase is no longer randomized by UseRandomizedStringHashAlgorithm.md

```diff
@@ -0,0 +1,31 @@
+## 115: AppDomainSetup.DynamicBase is no longer randomized by UseRandomizedStringHashAlgorithm
```

Comment:
{
  "Id": "115",
  "Title": "AppDomainSetup.DynamicBase is no longer randomized by UseRandomizedStringHashAlgorithm",
  "Details": "Prior to the .NET Framework 4.6, the value of AppDomainSetup.DynamicBase would be randomized between application domains, or between processes, if UseRandomizedStringHashAlgorithm was enabled in the app's config file. Beginning in the .NET Framework 4.6, AppDomainSetup.DynamicBase will return a stable result between different instances of an app running, and between different app domains. Dynamic bases will still differ for different apps; this change only removes the random naming element for different instances of the same app.",
  "Suggestion": "Be aware that enabling `UseRandomizedStringHashAlgorithm` will not result in `AppDomainSetup.DynamicBase` being randomized. If a random base is needed, it must be produced in your app's code rather than via this API.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "M:System.AppDomainSetup.get_DynamicBase",
    "P:System.AppDomainSetup.DynamicBase"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1008953",
  "Notes": "Should be easy to look for DynamicBase use while UseRandomizedStringHashAlgorithm is set\nSource analyzer status: Pri 3, Planned",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.AppDomainSetup.get_DynamicBase
- [x] P:System.AppDomainSetup.DynamicBase